docs/security/session_management.md

```diff
@@ -0,0 +1,59 @@
+# Session Management
+
+Session management is the process of handling user sessions to keep them secure, efficient, and continuous. It is used in applications that manage user authentication and authorization, such as web services, databases, and DBaaS platforms.
+
+## Percona Everest authentication methods
+
+Percona Everest supports two authentication methods:
+
+- Built-in authentication: Suitable for non-production environments
+
+- [External Identity Provider (IdP) authentication](Idp_integration.md): Suitable for production use and enables integration with systems such as Okta and Azure AD.
+
```

Comment:
The link to the `Idp_integration.md` does not work in the render, I assume it's because the PR is not merged yet, right?

Reply:
Yes.